### ISSUE TITLE
Add timestamps to log

### ISSUE THREAD

**Issue by blueyed** [anarcat/bup-cron#22]
The `logfile` should include timestamps.

**Comment by anarcat** [comment]
yeah, that would make sense... should be fairly easy to setup with a logfile-specific formatter...
